crates/swc_error_reporters/src/error.rs

```diff
@@ -0,0 +1,16 @@
+/// ParseSyntaxError is a custom error type that represents a syntax error in
+/// swc parse
+///
+/// We will return this error if swc parser occur syntax error, but we will
+/// ignore this error in rust side, because we will return this error in js
+/// side.
+#[derive(Debug)]
+pub struct ParseSyntaxError;
```

Comment:
This should have a more general name, I think?

Reply:
But not sure at the moment. I'm not sure if it's correct.

Reply:
how about `OnlyDiagnosticsError`? I'm not sure if it's suitable too.

Reply:
I think it would be better for now. (Mainly because I'm not sure about the naming at the moment)

Reply:
Got it. I update it right now.